DESCRIPTION

```diff
@@ -34,6 +33,8 @@ Suggests:
     devtools,
     ggplot2,
     knitr,
+    sf, 
+    geosphere,
```

Comment:
geosphere depends on sp, so really here we're adding 3 suggests (sp, sf, geosphere) rather than only 1 (sp). I'd prefer to keep with the sp package for now. sf/geosphere may be gaining in use and popularity, but I doubt that our  line of code will become redundant or obsolete anytime soon. I'd prefer the route of simplicity for now, and we can update it in the future if/when it becomes necessary.